### PR TITLE
Update main-nl.json

### PR DESCRIPTION
**Pull request by JohnProv**

### CODE REVIEW

File: lang/main-nl.json [hunk 1]
```diff
@@ -497,6 +497,7 @@
         "errorMissingPassword": "Voer alstublieft het wachtwoord van de vergadering in",
         "invalidPassword": "Ongeldig wachtwoord",
         "joinRejectedMessage": "Uw verzoek tot deelname is afgewezen door een moderator.",
+        "joinRejectedTitle": "Verzoek tot deelname afgewezen.",
```
Comment (saghul) [inline lang/main-nl.json:500]: I think this shouldn't end in a period

**Review by saghul — COMMENTED** [review]
LGTM with one nit.

File: lang/main-nl.json (hunk 1 above)
Comment (JohnProv) [reply lang/main-nl.json:500]: See main.json, there this string also ends with a period.
Should I remove this period?

Comment (saghul) [reply lang/main-nl.json:500]: Oh, yeah please do!

Comment (JohnProv) [reply lang/main-nl.json]: Removed!

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Dank je wel!

**Pull request merged**